src/test/java/hudson/plugins/gradle/WithGradleWorkFlowTest.java

```diff
@@ -44,19 +47,16 @@
     private String build_gradle = "writeFile(file:'build.gradle', text:'defaultTasks \\\'hello\\\'\\ntask hello << { println \\\'Hello\\\' }') \n";
 
     /*
-        TODO DO NOT MERGE: when jenkins-test-harness-tools 2.1 is released add GradleInstallation tool. Right now
-        these tests use the system's default Gradle. If Gradle is not installed these tests will fail.
-
-        TODO: convert these tests to Groovy OR create src/test/java/.../WithGradleWorkFlowTest.java
         TODO? add test for console annotation
         TODO? add test for reloading annotator on Jenkins restart
     */
 
     @Test
     public void testStepDefaultTools() throws Exception {
+        String name = ToolInstallations.configureDefaultGradle(new TemporaryFolder()).getName();
```

Comment:
We talked about using `docker-fixtures` for tests in this PR, though it is more work to set up.